brion/circuit.cpp

```diff
@@ -43,11 +43,12 @@ enum Section
     SECTION_CIRCUITSEEDS,
     SECTION_UNKNOWN
 };
+}
 
-class Circuit
+class Circuit::_Impl
```

Comment:
First time we do _Impl, I also don't see the point to deviate here. The Impl is by definition private, so _ would be correct, but it's also rather redundant. Change it everywhere else and I'm fine with it :)